docs/releases/1.10.txt

```diff
@@ -732,6 +732,17 @@ use the default_related_name ``bars``::
 
     >>> Foo.object.get(bars=bar)
 
+Custom related manager must define a ``_apply_rel_filters()`` method
+--------------------------------------------------------------------
+
+If you defined a custom manager class available through
+:meth:`~django.db.models.query.QuerySet.prefetch_related` you must make sure
+it defines a ``_apply_rel_filters()`` method.
+
+This method must accept a :class:`~django.db.models.query.QuerySet` instance
+as its single argument and return a version appropriately filetered for the
```

Comment:
a filtered version of the queryset...
(is this a private API such that we don't need to document it elsewhere?)